imagebuildah/build.go

```diff
@@ -95,6 +95,9 @@ type BuildOptions struct {
 	// specified, indicating that the shared, system-wide default policy
 	// should be used.
 	SignaturePolicyPath string
+	// SkipTLSVerify denotes whether TLS verification should or should not
```

Comment:
// SkipTLSVerify denotes whether TLS verification should not be used

Reply:
fixed